### DISCUSSION TITLE
[RFC] Merge Leaf API into Leaf MVC?

### DISCUSSION THREAD

**Discussion by mychidarko** [leafsphp/leaf#246] [Polls]
Leaf API was a project we started to provide a simple way to build APIs with Leaf. Over time, it became more and more like Leaf MVC and today it's just a stripped-down version of Leaf MVC. Maintaining two separate projects that do the same thing is not the best use of our resources, and we're considering merging Leaf API into Leaf MVC.

For users who use the Leaf CLI to generate their projects, this change will not affect you in any way. The CLI will just automatically set up your project to look exactly as Leaf API does today.

We would like to hear your thoughts on this change and how it might affect your projects.

**Comment by terrybr** [comment]
I believe this the right step to take.  On top of what's already mentioned, it can also create confusion for new users.

**Comment by mychidarko** [comment]
Thanks for all the input. We will go ahead with this change
